lib/utils.js

```diff
@@ -29,7 +29,10 @@ var assign = (exports.assign = require('object.assign').getPolyfill());
  * @throws {TypeError} if either constructor is null, or if super constructor
  *     lacks a prototype.
  */
-exports.inherits = util.inherits;
+exports.inherits = function(ctor, superCtor) {
+  // Wrapper prevents type declarations from referring to @types/node
```

Comment:
You could also just add a `/** @type {whatever} */` annotation onto this declaration to override the inferred type, instead of changing the implementation.